src/util/permissionIdentities.tsx

```diff
@@ -0,0 +1,191 @@
+import { LxdGroup, LxdIdentity } from "types/permissions";
+
+export type IdentitiesOrGroupsChangeSummary = Record<
+  string,
+  { added: Set<string>; removed: Set<string>; name: string }
+>;
+
+export const getIdentityIdsForGroup = (group: LxdGroup) => {
+  const oidcIdentityIds = group?.identities?.oidc || [];
+  const tlsIdentityIds = group?.identities?.tls || [];
+  const allIdentityIds = oidcIdentityIds.concat(tlsIdentityIds);
+  const totalIdentitiesCount = allIdentityIds.length;
+
+  return {
+    oidcIdentityIds,
+    tlsIdentityIds,
+    allIdentityIds,
+    totalIdentitiesCount,
+  };
```

Comment:
```suggestion
  return allIdentityIds;
```
I think all use cases can use this simpler return structure. The one were we want the count can be `getIdentityIdsForGroup(group).length`

Reply:
aah yes that's true, I was trying to cater for scenarios that's actually not needed :+1: